### PR TITLE
Add telegram-send recipe

### PR DESCRIPTION
**Pull request by proinsias**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by proinsias** [comment]
I confirm I am willing to be listed in the maintainer section.

**Comment by proinsias** [comment]
@conda-forge/help-python – I believe this is ready for review now. Not sure why AppVeyor builds are failing for these PRs.

File: recipes/telegram-send/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e6cda001a89e1cd02317fcd0456288a0cdd5aa56970dec6fbd83ad9af1db05ac
+
+build:
+  number: 0
+  entry_points:
+    - telegram-send=telegram_send:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<=34]
```
Comment (marcelotrevisani) [inline recipes/telegram-send/meta.yaml]:
```suggestion
  noarch: python
```

File: recipes/telegram-send/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - appdirs
+    - colorama
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/telegram-send/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/telegram-send/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - python-telegram-bot >=12.1.1
+  run:
+    - appdirs
+    - colorama
+    - python
```
Comment (marcelotrevisani) [inline recipes/telegram-send/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/telegram-send/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/telegram-send/meta.yaml]: I think you just need ``pip`` and ``python`` on ``host``. Could you please dele the other deps?

File: recipes/telegram-send/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - python-telegram-bot >=12.1.1
+
+test:
+  commands:
+    - telegram-send --help
+
+about:
+  home: "https://github.com/rahiel/telegram-send"
+  license: "GNU General Public v3 or later (GPLv3+)"
```
Comment (marcelotrevisani) [inline recipes/telegram-send/meta.yaml]:
```suggestion
  license: GPL-3.0
```

**Comment by proinsias** [comment]
@marcelotrevisani – I've updated as per your suggestions.

**Pull request merged**